pkg/kdump/kdump-view.jsx

```diff
@@ -357,7 +357,7 @@ export class KdumpPage extends React.Component {
             actions: [
                 {
                     clicked: this.handleApplyClick.bind(this),
```

Comment:
So this leaves behind `handleApplyClick`. In `pkg/networkmanager/dialogs-common.jsx` it leaves behind `id=...-apply`. These are just names so does not matter, just maybe would be a bit cleaner for future ourselves when we wonder why we call it "apply" :) `git grep "[aA]pply" pkg/` shows a lot of such cases. Not necessary to do anything, just pointing it out.

Reply:
Good point! I renamed these, at least all that I could find which immediately make sense.

I also adjusted cryptoPolicies, that landed recently and we didn't have it on the radar in this PR.